diff --git a/deploy/include/FSM/FSMState.h b/deploy/include/FSM/FSMState.h
index cbbc2af..ca1fba4 100644
--- a/deploy/include/FSM/FSMState.h
+++ b/deploy/include/FSM/FSMState.h
@@ -44,6 +44,64 @@ public:
             }
         }
 
+	// Keyboard transitions for simulation without a gamepad
+	if (state_string == "Passive")
+	{
+	    registered_checks.emplace_back(
+	        std::make_pair(
+	            []()->bool {
+	                return FSMState::keyboard &&
+	                       FSMState::keyboard->on_pressed &&
+	                       FSMState::keyboard->key() == "1";
+	            },
+	            FSMStringMap.right.at("FixStand")
+	        )
+	    );
+	}
+
+	if (state_string == "FixStand")
+	{
+	    registered_checks.emplace_back(
+	        std::make_pair(
+	            []()->bool {
+	                return FSMState::keyboard &&
+	                       FSMState::keyboard->on_pressed &&
+	                       FSMState::keyboard->key() == "2";
+	            },
+	            FSMStringMap.right.at("Velocity")
+	        )
+	    );
+	}
+
+        if (state_string == "Velocity")
+        {
+            registered_checks.emplace_back(
+                std::make_pair(
+                    []()->bool {
+                        return FSMState::keyboard &&
+                               FSMState::keyboard->on_pressed &&
+                               FSMState::keyboard->key() == "5";
+                    },
+                    FSMStringMap.right.at("Mimic_Dance1_subject2")
+                )
+            );
+        }
+
+        if (state_string == "Mimic_Dance1_subject2")
+        {
+            registered_checks.emplace_back(
+                std::make_pair(
+                    []()->bool {
+                        return FSMState::keyboard &&
+                               FSMState::keyboard->on_pressed &&
+                               FSMState::keyboard->key() == "3";
+                    },
+                    FSMStringMap.right.at("Velocity")
+                )
+            );
+        }
+
+
         // register for all states
         registered_checks.emplace_back(
             std::make_pair(
@@ -67,4 +125,4 @@ public:
     static std::unique_ptr<LowCmd_t> lowcmd;
     static std::shared_ptr<LowState_t> lowstate;
     static std::shared_ptr<Keyboard> keyboard;
-};
\ No newline at end of file
+};
